content/work/metecho.md

```diff
@@ -17,6 +21,74 @@ tasks:
   - Project Architecture
   - Front & Back-End Development
   - Lightning Design System React Implementation
+press:
+  - text: |
+      Working with OddBird was
+      **the best outsourcing experience I've ever had.**
+      It didn't feel like outsourcing at all.
+    <<: *jason
+    slug: outsourcing
+  - text: |
+      **OddBird became an extension of our team**
+      bringing UX, frontend, and backend brilliance
+      to take MetaDeploy from an idea into a production application.
```

Comment:
Should this quote only be on the MetaDeploy page instead of on both?

Reply:
Yep. Removed.